frontend/src/components/schedules/SchedulePaginator.vue

```diff
@@ -374,7 +374,7 @@ export default {
                 return false;
             } else {
                 // (big) TODO: This should also check the contents
-                return this.lastUsedClassSections.length != this.classSections.length;
+                return this.lastUsedClassSections != this.$store.getters.selectedClassSections;
             }
```

Comment:
Part of issue 4 is that this isn't checking if the user adds/deletes/updates an custom event to the generation. Consider this scenario:

1. User adds MATH 3A and ANTH 2
2. User generates schedules
3. User adds a custom event

`selectedClassSectionsHaveUpdated` is responsible for triggering new schedules to be generated when the inputs change, but since it's not (yet) looking at the custom events it won't trigger new schedules to be generated in the scenario above

Reply:
commit fef71ca resolved this comment